parsec-3.0/README.md

```diff
@@ -0,0 +1,15 @@
+##PARSEC 3.0
+###Available benchmarks:
+- **Parsec** : All except those belonging to  *netapps* package and "uptcpip" from *libs* package
```

Comment:
I looked at it and there is no "__mbstate_t" definition in the file they speak about.